src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costs = 40;
```

Comment:
The variable name 'costs' should be changed to 'costPerDay' for better readability.